### PR TITLE
Add master-infra and all-in-one node-configs

### PR DESCRIPTION
**Pull request by sdodson**
/cc @michaelgugino

### CODE REVIEW

**Comment by vrutkovs** [comment]
Lets verify all-in-one config by using it in `.papr.all-in-one.inventory`

File: roles/openshift_facts/defaults/main.yml [hunk 1]
```diff
@@ -130,4 +130,12 @@ openshift_node_groups:
     labels:
       - 'node-role.kubernetes.io/compute=true'
     edits: []
+  - name: node-config-master-infra
+    labels:
+      - 'node-role.kubernetes.io/infra=true,node-role.kubernetes.io/master=true'
+    edits: []
+  - name: node-config-all-in-one
```
Comment (michaelgugino) [inline roles/openshift_facts/defaults/main.yml]: I agree with @vrutkovs let's update the papr inventories to utilize this for CI.

**Comment by sdodson** [comment]
I think I'm going to go through and update the gcp jobs to use these as well.

**Pull request merged**